doc/reference.qbk

```diff
@@ -192,6 +185,10 @@
 
 [include generated/reverse.qbk]
 
+[section:similarity similarity]
+[include generated/similarity.qbk]
+[endsect]
+
```

Comment:
This doesn't work does it? Anyway, in the reference you should have both algorithms like it was before.